eternalcore-core/src/main/java/com/eternalcode/core/translation/implementation/PLTranslation.java

```diff
@@ -209,6 +209,11 @@ public static class PLTeleportSection implements TeleportSection {
         public Notice teleportedSpecifiedPlayerLastLocation = Notice.chat("<green>► <white>Przeteleportowano gracza <green>{PLAYER} <white>do ostatniej lokalizacji!");
         @Description(" ")
         public Notice lastLocationNoExist = Notice.chat("<red>✘ <dark_red>Nie ma zapisanej ostatniej lokalizacji!");
+
+        @Description(" ")
+        public Notice noPlayerToRandomTeleportFound = Notice.chat("<red>✘ <dark_red>Nie można odnaleźć gracza do teleportacji!");
+        @Description({ " ", "# {PLAYER} - Player to whom he was teleported" })
+        public Notice teleportedToRandomPlayer = Notice.chat("<green>► <white>Zostałeś losowo teleportowany do <green>{PLAYER}<white>!");
```

Comment:
Shouldn't this configuration option be in RandomTeleportSection?

Reply:
> I prefer to avoid the strong connection with random teleport (rtp), which is commonly referred to as random teleportation on a map. I would like to hear the viewpoints of other team members.

Yup
I think y right, that's why I proposed changing the command name here https://github.com/EternalCodeTeam/EternalCore/pull/546#discussion_r1349531874